src/main/java/frc/robot/RobotMap.java

```diff
@@ -2,5 +2,23 @@
 
 public class RobotMap {
 
+    public static final int PIGEON = 0;
+    public static final double ATTAINBLE_MAX_SPEED_MPS_SWERVE = 10;
+    public static final double SWEARVE_ABSOLUTE_FL_ZERO_ANGLE = 91.2;
+    public static final double SWEARVE_ABSOLUTE_FR_ZERO_ANGLE = 297.59;
+    public static final double SWEARVE_ABSOLUTE_RL_ZERO_ANGLE = 35.1;
+    public static final double SWEARVE_ABSOLUTE_RR_ZERO_ANGLE = 190;
+    public static final int SWERVE_LF_DRIVE_ID = 51;
+    public static final int SWERVE_RF_DRIVE_ID = 41;
+    public static final int SWERVE_LB_DRIVE_ID = 61;
+    public static final int SWERVE_RB_DRIVE_ID = 31;
+
+    public static final int SWERVE_LF_STEER_ID = 52;
+    public static final int SWERVE_RF_STEER_ID = 42;
+    public static final int SWERVE_LB_STEER_ID = 62;
+    public static final int SWERVE_RB_STEER_ID = 32;
+
+    public static final double DISTANCE_MID_TO_CON =0.37;
```

Comment:
```suggestion
    public static final double DISTANCE_MODULE_TO_CENTER_CHASSIS_METERS =0.37;
```
1. specify measurement unit: important to let others know which measurement units you are working with
2. the name is a bit confusing